docs/en/reference/working-with-associations.rst

```diff
@@ -500,8 +500,27 @@ and the "remove" operation.
         //...
     }
 
-Even though automatic cascading is convenient it should be used
-with care. Do not blindly apply cascade=all to all associations as
+Since ``cascade: persist`` is configured for the ``User#commentsAuthored``
+association, you can now create a user and persist his comment as follows:
+
+.. code-block:: php
+
+    <?php
+    $user = new User();
+    $myFirstComment = new Comment();
+    $user->addComment($myFirstComment);
+    
+    $em->persist($user);
+    $em->flush();
+
+.. note::
+
+    This code does not associate the newly created comment with the user.
+    To achieve this, you *always* have to call ``$myFirstComment->setUser($user);`` –
+    no matter if ``cascade: persist`` is set or not.
```

Comment:
> IMO what you're saying should be explained in the docs! 

Add it! :D

 > cascading as a *convenience feature.

Cascading **is** a convenience feature. It is not to be used as a default, but it is very commonly used when dealing with DDD aggregate roots.

 >  I think I'll somewhat rewrite the entire 8.6 chapter

Be my guest! Any improvements are hugely welcomed! 

 > I would explain the question "who should use it" at first, deferring all other topics to later. Reason: For a feature which is only useful for a small minority, it doesn't make sense to start with details.
 > 
 > Is that OK?

Absolutely!

 > However, one more question I have:
 > What exactly do you mean by "expose to the outside world"?

Take a `BankAccount` object (typical sad example): you can most probably deposit and withdraw from it, but you can't really modify its internals from the outside, as that would be an obvious violation of encapsulation. Inside it, you may have `Transaction` instances that aren't really visible to the outside world: all you want is for them to be saved/loaded correctly, but you don't want to give access to others.